wpilibc/athena/src/IterativeRobot.cpp

```diff
@@ -161,6 +162,20 @@ void IterativeRobot::TestInit() {
 }
 
 /**
+ * Periodic code for all modes should go here.
+ *
+ * Users should override this method for code which will be called periodically
+ * at a regular rate while the robot is in any mode.
+ */
+void IterativeRobot::RobotPeriodic() {
+  static bool firstRun = true;
+  if (firstRun) {
+    std::printf("Default %s() method... Overload me!\n", __FUNCTION__);
```

Comment:
I'm not sure we want this message here.  It will be generated for all existing robot programs, and basically mandates the template have yet another method.  Can we just make it an empty body?